### DISCUSSION TITLE
Inquire how to put hyperlinks in comments on the project details page

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4936] [Plugins]
![스크린샷 2024-06-26 175005](https://github.com/kimai/kimai/assets/165639563/a28f45f6-ec62-4073-a539-a9db6b4a656c)

I want to put a hyperlink to the comments on the project details page, how can I do it?

**Comment by seleondevel** [comment]
@kevinpapst 

kimai/templates/project/details.html.twig 

line23

 {% if project.comment is not empty %}
      <div class="comment p-3">
          {{ project.comment|comment2html(true) }}
      </div>
  {% endif %}

What is comment2html (true)?

Can you tell me how to use it?

**Comment by kernie** [comment]
Hi,

I think it is finally running this:

![grafik](https://github.com/kimai/kimai/assets/23321751/cee9936e-1d7d-4209-be2a-bb00971e7b55)

And if Markdown is allowed there (thus the slider is enabled)

![grafik](https://github.com/kimai/kimai/assets/23321751/20ac85fc-3b19-4693-ad27-25deb543addc)

it  would expect a Markdown formatted link like 

```
[Link text Here](https://link-url-here.org)
```
Just an assumption, though. I didn't test it.

EDIT: I tested it, and it works.

**Reply by seleondevel** [reply]
Wow!
I didn't know this cool feature was on the Kimai settings menu.
Thank you for sharing your knowledge.
Thanks to you, I solved it well!
